environments/backfillStorage.toml

```diff
@@ -0,0 +1,196 @@
+[exporter]
+    home     = "github.com/makerdao/vulcanizedb"
+    name     = "transformerExporter"
+    save     = false
+    transformerNames = [
+        "knc_flip",
+        "tusd_flip",
+        "usdc_a_flip",
+        "usdc_b_flip",
+        "wbtc_flip",
+        "zrx_flip",
+        "median_knc",
```

Comment:
one of us is going to need to rebase to get `median_mana` added to this list

![image](https://user-images.githubusercontent.com/7875345/88705213-d1d0fd00-d0d4-11ea-9b01-e7378ae3817e.png)